docs/tutorials/track-trace-ledger-api/track-trace-ledger-api-tutorial-102.md

```diff
@@ -1,4 +1,4 @@
-# IOTA Track & Trace Ledger APIs 102 Tutorial : 
+# IOTA Track & Trace Ledger APIs 102 Tutorial : Device Events Recorded on the Tangle.
```

Comment:
Better "Consuming Events" - but why change this?

Reply:
All the other sections have a clarification after the tutorial number.  I thought it'd be good to be consistent.